addons/point_of_sale/security/ir.model.access.csv

```diff
@@ -36,3 +36,5 @@ access_barcode_rule_pos_user,barcode.rule.pos.user,barcodes.model_barcode_rule,g
 access_barcode_rule_pos_manager,barcode.rule.pos.manager,barcodes.model_barcode_rule,group_pos_manager,1,1,1,1
 access_bank_statement_cashbox_user,account.bank.statement.cashbox user,account.model_account_bank_statement_cashbox,group_pos_user,1,1,1,0
 access_account_cashbox_line_user,account.cashbox.line user,account.model_account_cashbox_line,group_pos_user,1,1,1,0
+access_pos_payment_user,pos.payment user,model_pos_payment,group_pos_user,1,1,1,1
+access_pos_payment_method_user,pos.payment.method user,model_pos_payment_method,group_pos_user,1,1,1,1

```

Comment:
not sure the ppos user should be able to create payment method. Only the manager for me.